index.html

```diff
@@ -191,7 +190,28 @@ <h2 class="heading1">Featured Speakers</h2>
         <button class="more">See more <i class="fa fa-arrow-down" aria-hidden="true"></i></button>
       </div>
     </section>
+
+    <section class="partner" id="partners">
+      <div class="heading-group">
+        <h3 class="heading1">Partners</h3>
+        <hr class="heading-line">
+      </div>
+
+      <ul>
+        <li><img src="images/logoestamuycool.png" alt="estamuy cool logo"></li>
+        <li><img src="images/mask_creativa_logo.png" alt="mask creative logo"></li>
+      </ul>
+    </section>
```

Comment:
- [ ] _[Optional]_ This part of your index page (on desktop view) does not follow the design on Behance. It seems you didn't work on that part of the desktop view. I'll suggest you make the change as it'll make your page look much better 


Yours | Behance
----- | -----
![image](https://user-images.githubusercontent.com/16058470/124996071-40863c80-e040-11eb-8803-4de9814056f5.png) | ![image](https://user-images.githubusercontent.com/16058470/124996129-598eed80-e040-11eb-9017-2ca73dee0bde.png)